onadata/apps/logger/management/commands/change_s3_media_permissions.py

```diff
@@ -5,29 +5,29 @@
 
 from django.core.management.base import BaseCommand, CommandError
 from django.core.files.storage import get_storage_class
-from django.utils.translation import ugettext as _, ugettext_lazy
+from django.utils.translation import gettext as t
```

Comment:
As per our discussion, you can remove gettext in management commands. 
Please review all the management commands since I've spotted this import at few other places.